### PR TITLE
the home page wasn't finding the other files so I corrected that

### PR DESCRIPTION
**Pull request by osa711**
I also made a progress with the vehicles page

### CODE REVIEW

File: HTML/Vehicles.html [hunk 1]
```diff
@@ -1,8 +1,128 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-</head>
-<body bgcolor = "violet">
-<h1> This is the Vehicles page </h1>
+    <style>
```
Comment (jr7072) [inline HTML/Vehicles.html]: Would want this on an external css files. Especially since we have a folder dedicated to it

File: HTML/Vehicles.html [hunk 2]
```diff
@@ -1,8 +1,128 @@
...
 <head>
-</head>
-<body bgcolor = "violet">
-<h1> This is the Vehicles page </h1>
+    <style>
+        .pageHeader{
+            text-align: center;
+        }
+        .carRow{
+            display: flex;
```
Comment (jr7072) [inline HTML/Vehicles.html]: For centering you could also take advantage of flex box properties. Such as align-items to center vertically and justify-content to center horizontally. Margin auto is good from experience it's usually never works with some additional properties. align-items: center; justify-content: center should work and it does not break as easily.

File: HTML/Vehicles.html [hunk 3]
```diff
@@ -1,8 +1,128 @@
...
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+        <div class="carListing 4">
+            <img src="../Images/placeHolder.png" height="200" width="200">
+            <h2 class="carName"> Car name</h2>
+            <h2 class="carPrice"> 20000</h2>
+        </div>
+    </div>
```
Comment (jr7072) [inline HTML/Vehicles.html:104]: The structure of the car rows is perfect!

**Comment by jr7072** [comment]
Added everyone else as second reviewer. Looks good just needs some styling changes.

File: HTML/Vehicles.html (hunk 1 above)
Comment (osa711) [reply HTML/Vehicles.html]: yeah I'll do that, sorry I didn't notice the folder

File: HTML/Vehicles.html (hunk 2 above)
Comment (osa711) [reply HTML/Vehicles.html]: ah ok, I'll change that then

File: HTML/Vehicles.html (hunk 3 above)
Comment (osa711) [reply HTML/Vehicles.html:104]: thanks

File: HTML/Vehicles.html (hunk 1 above)
Comment (jr7072) [reply HTML/Vehicles.html]: No worries!

File: HTML/Vehicles.html (hunk 2 above)
Comment (jr7072) [reply HTML/Vehicles.html]: Yeah its pretty cool once you know about flexbox I use it all the time. There's also display: grid but that is very tricky to use in my opinion

File: CSS/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    text-align: center;
+}
+.carRow{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 10%;
+}
+.carRow div{
+
```
Comment (jr7072) [inline CSS/stylesheet.css:13]: white space is here too

**Review by jr7072 — CHANGES_REQUESTED** [review]

**Comment by jr7072** [comment]
Sorry for something that seems annoying. But it keeps the code nice and neat and readable in the long run.

**Comment by osa711** [comment]
> Sorry for something that seems annoying. But it keeps the code nice and neat and readable in the long run.

it's fine, it does improve the readability of the code.

**Review by jr7072 — APPROVED** [review]
Looks all good with me. Just needs another person to approve it.

**Comment by osa711** [comment]
> Looks all good with me. Just needs another person to approve it.

neat

**Pull request merged**